homeassistant/components/redgtech/config_flow.py

```diff
@@ -0,0 +1,46 @@
+from homeassistant import config_entries
+import voluptuous as vol
+import logging
+from .const import DOMAIN
+from redgtech_api import RedgtechAPI
+
+_LOGGER = logging.getLogger(__name__)
+
+class RedgtechConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config Flow for Redgtech integration."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial user step for login."""
+        errors = {}
+        
+        if user_input is not None:
+            email = user_input.get("email")
+            password = user_input.get("password")
```

Comment:
```suggestion
            email = user_input.get(CONF_EMAIL)
            password = user_input.get(CONF_PASSWORD)
```
we use consts for those things `from homeassistant.const`